### ISSUE TITLE
Error Responses are Not Indexed or Searchable or are Pulled from Another Repository

### ISSUE THREAD

**Issue by googlier** [hirosystems/clarinet#699]
Clarinet seems to reference error codes that do not exist inside of the clarinet repository.  This makes debugging error codes a nightmare.

Error publishing transactions: unable to post transaction
Message("{\"error\":\"transaction rejected\",\"reason\":\"ConflictingNonceInMempool\
https://github.com/search?q=ConflictingNonceInMempool&type=code

**Comment by lgalabru** [comment]
Indeed, Clarinet involves other components like stacks-node, bitcoind, and many other libraries.
In the link that you provided, the answer to your question is, literally, the 1st link.
Conflicting nonce in mempool means that you submitted a transaction with 2 identical nonces.
